src/common/pkcs11/pkcs11.cpp

```diff
@@ -259,30 +259,82 @@ Error GenPIN(String& pin)
     return ErrorEnum::eNone;
 }
 
+#if defined(AOS_CONFIG_PKCS11_USE_STATIC_LIB)
```

Comment:
#include <dlfcn.h> should be excluded in case AOS_CONFIG_PKCS11_USE_STATIC_LIB is used: zephyr doesn't have it.

Reply:
done